rlclientlib/rlclientlib.vcxproj.filters

```diff
@@ -136,7 +143,9 @@
     <None Include="schema\v2\CaEvent.fbs" />
     <None Include="schema\v2\MultiSlotEvent.fbs" />
     <None Include="schema\v2\Event.fbs" />
-    <None Include="schema\v2\DedupInfo.fbs" />
+    <None Include="ClassDiagram.cd" />

```

Comment:
Was this intentionally committed?

Reply:
Autogenerated. Getting rid of ClassDiagram.cd.